plugin/index.ts

```diff
@@ -0,0 +1,1078 @@
+import * as BabelCore from '@babel/core';
+import * as BabelTypes from '@babel/types';
+import generate from '@babel/generator';
+import traverse from '@babel/traverse';
```

Comment:
We need to add these dependencies to `devDependencies` in `package.json` (project root).